app/app.go

```diff
@@ -781,19 +801,21 @@ func NewStrideApp(
 	// - staketia
 	// - stakedym
 	// - ratelimit
+	// - ibchooks
```

Comment:
re: ordering - I think memo data should trigger one of autopilot / ibchooks / pfm, but should not trigger 2 or 3 of them. Is this also your understanding?

If so, I don't think the order of those 3 modules in the stack matters

Reply:
I think today we don't have any use cases of all 3 at once. In the future, I could see us using autopilot + ibchooks together. And then I think there should never be a use case for (autopilot + pfm) or (ibchooks + pfm).

The only current restriction in the code is that packets can't be autopilot and pfm. Feels safest rn to just extend it to only 1 of the 3

Reply:
[done](https://github.com/Stride-Labs/stride/pull/1173/commits/f00f7c1bf1594f6e19cd479a5951bc10b1efbb7d)

Reply:
Nice!